tests/FSharp.Data.Fred.Tests/UnitTest1.fs

```diff
@@ -1,11 +1,155 @@
+
+// Tests
+
+// Tests on myFred.Series.Observations
+
+[<Test>] 
+let ``Test 1961-04-01 GS10 observation`` () =  
+    GS10obs.Value |> should (equalWithin tolerance) 3.78
+
+[<Test>] 
+let ``Test 2015-04-03 DTP10J25 observation`` () =  
+    DTP10J25obs.Value |> should (equalWithin tolerance) 0.089
+
+// Tests on myFred.Series.Categories
+
+[<Test>] 
+let ``Test GS10 category id field`` () =  
+    GS10Categories |> Array.map(fun x -> x.Id) |> fun x -> x.[0] |> should (equalWithin tolerance) 115
+
+[<Test>] 
+let ``Test GS10 category name field`` () =  
+    GS10Categories |> Array.map(fun x -> x.Name) |> fun x -> x.[0] |> should equal "Treasury Constant Maturity"
+
+[<Test>] 
+let ``Test GS10 category parentId field`` () =  
+    GS10Categories |> Array.map(fun x -> x.ParentId) |> fun x -> x.[0] |> should (equalWithin tolerance) 22
+
+[<Test>] 
+let ``Test DTP10J25 category id field`` () =  
+    DTP10J25Categories |> Array.map(fun x -> x.Id) |> fun x -> x.[0] |> should (equalWithin tolerance) 82
+
+[<Test>] 
+let ``Test DTP10J25 category name field`` () =  
+    DTP10J25Categories |> Array.map(fun x -> x.Name) |> fun x -> x.[0] |> should equal "Treasury Inflation-Indexed Securities"
+
+[<Test>] 
+let ``Test DTP10J25 category parentId field`` () =  
+    DTP10J25Categories |> Array.map(fun x -> x.ParentId) |> fun x -> x.[0] |> should (equalWithin tolerance) 22
+
+// Tests on myFred.Series.Info
+
+[<Test>] 
+let ``Test GS10 Info Title field`` () =  
+    GS10Info |> Array.map(fun x -> x.Title) |> fun x -> x.[0] |> should equal "10-Year Treasury Constant Maturity Rate"
+
+[<Test>] 
+let ``Test GS10 Info Frequency field`` () =  
+    GS10Info |> Array.map(fun x -> x.Frequency) |> fun x -> x.[0] |> should equal "Monthly"
+
+[<Test>] 
+let ``Test GS10 Info Units field`` () =  
+    GS10Info |> Array.map(fun x -> x.Units) |> fun x -> x.[0] |> should equal "Percent"
+
+[<Test>] 
+let ``Test GS10 Info SeasonalAdjustment field`` () =  
+    GS10Info |> Array.map(fun x -> x.SeasonalAdjustment) |> fun x -> x.[0] |> should equal "Not Seasonally Adjusted"
+
+[<Test>] 
+let ``Test DTP10J25 Info Title field`` () =  
+    DTP10J25Info |> Array.map(fun x -> x.Title) |> fun x -> x.[0] |> should equal "10-Year 0-1/4% Treasury Inflation-Indexed Note, Due 1/15/2025"
+
+[<Test>] 
+let ``Test DTP10J25 Info Frequency field`` () =  
+    DTP10J25Info |> Array.map(fun x -> x.Frequency) |> fun x -> x.[0] |> should equal "Daily"
+
+[<Test>] 
+let ``Test DTP10J25 Info Units field`` () =  
+    DTP10J25Info |> Array.map(fun x -> x.Units) |> fun x -> x.[0] |> should equal "Percent"
+
+[<Test>] 
+let ``Test DTP10J25 Info SeasonalAdjustment field`` () =  
+    DTP10J25Info |> Array.map(fun x -> x.SeasonalAdjustment) |> fun x -> x.[0] |> should equal "Not Seasonally Adjusted"
+
+// Tests on myFred.Series.Release
+
+[<Test>] 
+let ``Test GS10 Release Id field`` () =  
+    GS10Release |> Array.map(fun x -> x.Id) |> fun x -> x.[0] |> should (equalWithin tolerance) 18
```

Comment:
This is fine, no need to change, but for future reference it is more idiomatic to use `Array.head` or `Array.item 0`
```fsharp
GS10Release
|> Array.map(fun x -> x.Id)
|> Array.head
|> should (equalWithin tolerance) 18
```